package.json

```diff
@@ -9,39 +9,70 @@
+    "f:test": "mocha -r test/tshook.js test/**/*.ts || true",
+    "prebuild": "tsc -t es5 -m es6 --skipLibCheck -p ./tsconfig.json",
+    "build": "npm-run-all --parallel build:dev build:prod",
+    "build:dev": "NODE_ENV=development ./node_modules/.bin/rollup -c",
+    "build:prod": "NODE_ENV=production ./node_modules/.bin/rollup -c",
+    "build:watch": "NODE_ENV=watch yarn run tsc --watch -p .",
+    "coverage": "npm run pretest && node --harmony ./node_modules/istanbul/lib/cli.js cover --root ./build/src --report lcov --report text ./node_modules/mocha/bin/_mocha ./build/test/**/*.js",
+    "clean": "rimraf ./dist ./build ./coverage ./lib",
+    "test": "NODE_ENV=test mocha --opts test/mocha.opts",
+    "test:jsdom": "NODE_ENV=test mocha --require test/setup/setup-dom.js 'test/**/*.test.ts'",
+    "test:watch": "yarn run test --watch",
+    "debug": "npm-run-all clean pretest",
+    "lint": "npm-run-all --parallel lint:src lint:tests",
+    "lint:src": "tslint src/**/*.ts src/**/*.tsx ",
+    "lint:tests": "tslint test/**/*.test.ts test/**/*.test.tsx",
+    "update:dependencies": "ncu -u"
   },
   "devDependencies": {
-    "@types/chai": "^3.5.0",
-    "@types/chai-as-promised": "0.0.30",
-    "@types/chai-things": "0.0.32",
+    "@types/chai": "^4.0.4",
+    "@types/chai-as-promised": "^7.1.0",
+    "@types/chai-things": "^0.0.32",
     "@types/lodash": "^4.14.62",
-    "@types/mocha": "^2.2.40",
-    "@types/node": "^7.0.12",
+    "@types/lodash.snakecase": "^4.1.3",
+    "@types/mocha": "^2.2.44",
+    "@types/node": "^8.0.58",
+    "@types/sinon": "^4.1.2",
+    "@types/sinon-chai": "^2.7.29",
     "awesome-typescript-loader": "^3.0.4-rc.2",
     "babel-plugin-external-helpers": "^6.22.0",
     "babel-preset-env": "^1.4.0",
     "babel-preset-es2015": "^6.24.1",
-    "chai": "^3.5.0",
-    "chai-as-promised": "^6.0.0",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
     "chai-things": "^0.2.0",
     "fetch-mock": "^5.10.0",
     "isomorphic-fetch": "^2.2.1",
     "lodash.clonedeep": "^4.5.0",
     "lodash.snakecase": "^4.1.1",
-    "mocha": "^3.2.0",
+    "mocha": "^4.0.1",
+    "rimraf": "2.6.2",
+    "rollup": "0.52.1",
+    "rollup-plugin-buble": "0.18.0",
+    "rollup-plugin-commonjs": "8.2.6",
+    "rollup-plugin-copy": "0.2.3",
+    "rollup-plugin-replace": "2.0.0",
+    "rollup-plugin-typescript2": "0.8.4",
+    "rollup-plugin-uglify": "2.0.1",
     "shx": "^0.2.2",
-    "sinon": "^2.1.0",
-    "ts-node": "^3.0.2",
-    "typescript": "^2.1.6",
-    "webpack": "^2.2.1",
+    "sinon": "^4.0.2",
+    "sinon-chai": "2.14.0",
+    "ts-node": "4.0.1",
+    "tslint": "5.8.0",
+    "typescript": "^2.6.2",
+    "webpack": "^3.10.0",
     "winston": "^2.3.1"
   },
   "dependencies": {
+    "@types/inflected": "1.1.29",
     "es6-promise": "^4.0.5",
+    "inflected": "2.0.3",
     "lodash-es": "^4.17.4",
+    "npm-check-updates": "^2.13.0",
```

Comment:
Seems like `npm-check-updates` and `@types/inflected` at least should be devDependencies instead of dependencies?